content/spin/v1/serverless-ai-hello-world.md

```diff
@@ -138,7 +138,7 @@ This is what your `spin.toml` file should look like, based on whether you’re u
  
```

Comment:
It won't let me comment but I noticed line 123 should be "Configuring" not "Configuration" I believe